docker-neo4j/README.md

```diff
@@ -0,0 +1,23 @@
+# Deployment of Neo4J Container with Docker Compose
+
+Built by: [jmspring](https://github.com/jmspring)
+
+This template allows you to deploy an Ubuntu Server 15.04 VM with Docker (using the [Docker Extension][ext])
+and starts a Neo4J instance listening on ports 7474 (non-ssl) and 7473 (ssl).  The data disk
+for the Neo4J instance is an external disk mounted on the VM.  The container is created 
+using the [Docker Compose][compose] capabilities of the [Azure Docker Extension][ext].
+
+Below are the parameters that the template expects:
```

Comment:
Can remove these parameters, we'll soon start rendering this readme on Azure.com and parameters are already rendered there automatically so this ends up being a duplicate